tools/build_defs/repo/http.bzl

```diff
@@ -191,6 +191,8 @@ def _http_file_impl(ctx):
     return _update_integrity_attr(ctx, _http_file_attrs, download_info)
 
 _HTTP_JAR_BUILD = """\
+load("@rules_java//java:java_import.bzl", "java_import")
```

Comment:
Could you load `http_jar` from `rules_java` instead? Then all this code can be deleted from bazel_tools.